### PR TITLE
[AIRFLOW-XXXX] Consistency fixes in new documentation

### PR DESCRIPTION
**Pull request by potiuk**
---
Issue link: `Document only change, no JIRA issue`

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -341,16 +341,16 @@ itself comes bundled with jQuery and bootstrap. While they may be phased out
...
+Install yarn and Its Packages
+-----------------------------
 
 Make sure yarn is available in your environment.
 
-To install it on macOS:
+To install yarn on macOS:
 
-1.  Run the following commands (taken from `this source <https://gist.github.com/DanHerbert/9520689>`__):
+1.  Run the install commands (taken from `this source <https://gist.github.com/DanHerbert/9520689>`__):
```
Comment (kaxil) [inline CONTRIBUTING.rst]:
```suggestion
1.  Run the following commands (taken from `this source <https://gist.github.com/DanHerbert/9520689>`__):
```

Let's keep "following" as it has a command to set config too

**Review by kaxil — COMMENTED** [review]
Mostly LGTM, just a minor comment.

There is also a merge conflict

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -155,16 +155,16 @@ these guidelines:
 Airflow Git Branches
 ====================
 
-All new development in Airflow happens in ``master`` branch. All PRs should target that branch.
-We also have ``v1-10-test`` branch which is used to test ``1.10.x`` series of Airflow and where committers
+All new development in Airflow happens in the ``master`` branch. All PRs should target that branch.
+We also have a ``v1-10-test`` branch that is used to test ``1.10.x`` series of Airflow and where committers
 (and only committers) cherry-pick selected commits from the master branch.
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
cherry-pick selected commits from the master branch.
```
Just feels better imho.

**Comment by kaxil** [comment]
Mergeable fails :)

❌ Must include description for the PR
Input : ---
Issue link: Document only change, no JIRA issue

**Review by mik-laj — APPROVED** [review]

**Pull request merged**